main/src/hadd.cxx

```diff
@@ -1,46 +1,49 @@
-/*
-
-  This program will add histograms (see note) and Trees from a list of root files and write them
-  to a target root file. The target file is newly created and must not be
+/**
```

Comment:
Add author and copyright header here?

Reply:
Maybe, but I wouldn't know which one. I just want to make this things appear in the docs. :-)

Reply:
The authors are at the end of this comment section line 99, I think they will appear in the docs anyway but it'd be better with the right \author doxygen command.